base/sparse/cholmod.jl

```diff
@@ -1541,6 +1541,88 @@ ldltfact(A::Union{SparseMatrixCSC{T},SparseMatrixCSC{Complex{T}},
     Hermitian{T,SparseMatrixCSC{T,SuiteSparse_long}}};
     kws...) where {T<:Real} = ldltfact(Sparse(A); kws...)
 
+## Rank updates
+
+"""
+    lowrankupdowndate!(F::Factor, C::Sparse, update::Cint)
+
+Update an `LDLt` or `LLt` Factorization `F` of `A` to a factorization of `A±C*C'`.
+
+If sparsity preserving factorization is used, i.e. `L*L'*=P*A*P'` then the new
```

Comment:
is there an extra `*` in this after the `L'` ? spaces around the operators might make some of these easier to read